src/fal/logger.py

```diff
@@ -0,0 +1,53 @@
+from dataclasses import dataclass
+
+from dbt.logger import log_manager
+from dbt.events.functions import fire_event
```

Comment:
Can we rename this function when we use it? I think calling `fire_event` each time we want to log is a bit awkward.